rasa/nlu/featurizers/count_vectors_featurizer.py

```diff
@@ -158,28 +185,37 @@ def __init__(
         self._check_analyzer()
 
         # declare class instance for CountVectorizer
-        self.vectorizer = vectorizer
-
-    def _get_message_text(self, message):
-        if message.get("spacy_doc"):  # if lemmatize is possible
-            text_tokens = [t.lemma_ for t in message.get("spacy_doc")]
-            # text = " ".join([t.lemma_ for t in message.get("spacy_doc")])
-        elif message.get("tokens"):  # if directly tokens is provided
-            text_tokens = [t.text for t in message.get("tokens")]
+        if vectorizer:
```

Comment:
I don't think this makes a lot of sense as this won't set the `self.vectorizer` attribute in the `None` case. What should happen if there is no vectorizer?

Reply:
Removed now. I had added it for checking if it resolves the pytype check, but it works after removing it too.